components/atom/upload/src/index.js

```diff
@@ -105,7 +133,21 @@ AtomUpload.propTypes = {
   status: PropTypes.oneOf(Object.values(STATUSES)).isRequired,
 
   /** Callback to be called (with files selected) when there`s a file selection (via click or drag & drop) */
-  onFilesSelection: PropTypes.func
+  onFilesSelection: PropTypes.func,
+  /**
+   * Allow drag 'n' drop (or selection from the file dialog) of multiple files
+   */
+  multiple: PropTypes.bool,
+  /** Maximum file size (in bytes) */
+  maxSize: PropTypes.number,
+  /**
+   * Set accepted file types.
+   * See https://github.com/okonet/attr-accept for more information.
+   */
+  accept: PropTypes.oneOfType([
```

Comment:
format?

Reply:
accept is the same prop that the wrapped library uses, so for the sake of homogeneity I would keep it as accept.